app/controllers/assignments_controller.rb

```diff
@@ -582,6 +586,10 @@ def download_config_files
         yml_criteria = assignment.criteria.reduce({}) { |a, b| a.merge b.to_yml }
         f.write yml_criteria.to_yaml
       end
+      zipfile.get_output_stream(CONFIG_FILES[:annotations]) do |f|
+        f.write convert_to_yml(assignment.annotation_categories)
+      end
+      assignment.starter_file_config_to_zip(zipfile, 'starter-file-config-files', CONFIG_FILES[:starter_files])
```

Comment:
`'starter-file-config-files'` should also be a string in CONFIG_FILES (avoid using [magic strings](https://en.wikipedia.org/wiki/Magic_string))